src/md/src/WorkWithReports/FilteringLaunches.md

```diff
@@ -29,7 +29,8 @@ This filter has next conditions: contains, not contains, equals, not equals.
 values are available:
 
 -   **Tags** - 1 symbol is enough for this search.
-The system searches for launches with the specified tag(s). Several tags may be specified. This filter has next conditions: AND/OR. 
+The system searches for launches with the specified tag(s) or without specified tag(s). Several tags may be specified. This filter has next conditions: ALL/ANY and WITHOUT ALL/WITHOUT ANY. 
```

Comment:
for lanches with or without specified tag(s). Multiple tags may be specified. The following filtering conditions are available: ....